lib/util/support.js

```diff
@@ -4,6 +4,7 @@
 const { guessBrowser, support: isWebRTCSupported } = require('@twilio/webrtc/lib/util');
 
 const SUPPORTED_CHROME_BASED_BROWSERS = [
+  'crios',
```

Comment:
"Chrome on iPhone" qualifies as `rebrandedChrome`, so I had to add support for it in this array. See line 167:
```js
const rebrandedChrome = rebrandedChromeBrowser(browser);
```

And this part of the final conditional on line 147:
```js
(!rebrandedChrome || SUPPORTED_CHROME_BASED_BROWSERS.includes(rebrandedChrome))
```